crypto/fipsmodule/modes/asm/ghash-x86_64.pl

```diff
@@ -114,10 +117,27 @@
 
 $do4xaggr=1;
 
+# common register layout
+$nlo="%rax";
+$nhi="%rbx";
+$Zlo="%r8";
+$Zhi="%r9";
+$tmp="%r10";
+$rem_4bit = "%r11";
+
+$Xi="%rdi";
+$Htbl="%rsi";
+
+
```

Comment:
I added back this chunk to stay in sync with BoringSSL. I believe it is dead code and I left a comment on the BoringSSL CL to let them know that it is dead code. I hope they will remove it in the future.